helix-view/src/editor.rs

```diff
@@ -74,6 +75,52 @@ where
     )
 }
 
+#[derive(Serialize, Deserialize, Default)]
+#[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
+struct LayoutMap {
+    from: String,
+    into: String,
+}
```

Comment:
I'm very opposed to this representation. It's a lot of indirection two have a `Vec<LayoutMap>` turn into a `HashMap<char, char>` when we could just use a TOML table. This is less verbose than a TOML table but we don't really need to optimize for concise config - I would rather have it be explicit.